### PR TITLE
handle case of concatenated modules which have a null context

### PR DESCRIPTION
**Pull request by winstonewert**

### CODE REVIEW

File: test/fixtures/app.js [hunk 1]
```diff
@@ -1,5 +1,3 @@
-import getUrls from 'get-urls';
+import {process} from './lib'
```
Comment (SamVerschueren) [inline test/fixtures/app.js]: Do not change this file, rather create a new one.

File: test/test.js [hunk 2]
```diff
@@ -2,10 +2,11 @@ import test from 'ava';
 import webpack from 'webpack';
 import pify from 'pify';
 import options from './fixtures/webpack.config';
+import optionsConcat from './fixtures/webpack.concat.config';
```
Comment (SamVerschueren) [inline test/test.js]: I believe you forgot to push this file.

Comment (winstonewert) [reply test/test.js]: doi!

File: test/fixtures/webpack.concat.config.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+	entry: path.join(__dirname, '/app.concat.js'),
+	output: {
+		path: __dirname,
+		filename: 'bundle.js'
+	},
+	plugins: [
+		new BabelEnginePlugin({
+			presets: ['env']
+		}),
+        new webpack.optimize.ModuleConcatenationPlugin(),
```
Comment (SamVerschueren) [inline test/fixtures/webpack.concat.config.js]: The indentation seems to be incorrect.

File: test/fixtures/webpack.concat.config.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+		new UglifyJSPlugin(),
```
Comment (SamVerschueren) [inline test/fixtures/webpack.concat.config.js]: Remove trailing comma.

File: test/fixtures/webpack.concat.config.js (hunk 3 above)
Comment (winstonewert) [reply test/fixtures/webpack.concat.config.js]: tabs vs spaces issue. fixed.

File: test/fixtures/webpack.concat.config.js (hunk 4 above)
Comment (winstonewert) [reply test/fixtures/webpack.concat.config.js]: fixed.

**Comment by winstonewert** [comment]
So I did run the test locally, and they passed. So I'm confused why travis is angry at me.

It appears to be complaining about formatting, but its complaining about one line I didn't touch and another file I didn't touch. I also can't see what is complaint is since the formatting likes ok to me.

**Comment by winstonewert** [comment]
Ok, figured it out. Since I originally made the pull request and now xo has pushed an upgraded version which was more strict about blank lines at the start of class definitions. 

I've pinned the xo version so that the tests don't get randomnly broken again and fixed the complaints.

**Comment by thasmo** [comment]
@SamVerschueren Tests pass. Does it look good to you? Thanks!

**Comment by winstonewert** [comment]
For what its worth, I gave up on using this project after a month with no response on my pull request.

I now simply apply the babel-loader to everything in my node_modules directory when doing a production build.

**Comment by SamVerschueren** [comment]
Thanks for giving maintainers the time to respond. People have personal things besides dealing with opensource stuff. Like having to deal with a newborn baby...
PRs are easily lost between all the GitHub notifications. There's no harm done if you would've pinged me like @thasmo did.

Besides that, if the solution you are using right now works for you, fine. There are plenty of ways to deal with the problem this plugin tries to solve.

I'm merging and releasing this PR the first chance I get when I'm on my computer.

**Comment by winstonewert** [comment]
Sorry, I didn't mean to come across as complaining! Certainly, I have no basis for complaining about what other people provide to me for free! 

I did conclude that the project was abandoned, no commits for 11 months, no response to the pull request, half a year old unresolved issue. It didn't occur to me to ping you, I'll keep that in mind for future.

I just wanted to share an alternative that worked for me, in case anyone else found it useful.

**Comment by SamVerschueren** [comment]
Released in `0.2.0`. Thanks for the PR!

**Pull request merged**